ViAn/GUI/reportgenerator.cpp

```diff
@@ -170,12 +167,12 @@ std::string ReportGenerator::date_time_generator() {
  * The name will be based on the project and current date and time.
  * @param activeDocument, the document that is selected
  */
```

Comment:
I know it's a comment on the other pull request but active_Document should still be sanke_case, just fix it here in case you are faster.
Also change the comment to match